rss-reader/Model/Entity/Channel/FeedChannel.m

```diff
@@ -26,7 +26,7 @@ @implementation FeedChannel
 
 + (instancetype)objectWithDictionary:(NSDictionary *)dictionary {
     if(!dictionary) {
-        @throw NSInvalidArgumentException;
+        assert(NO);
         return nil;
```

Comment:
In this case, same as throw an exeption, you can't return nil object for nil dictionary in Debug configuration. If you expect that in Release - probably ok, if you everytime check for non nil. But I think that more preferable to create non failable initializer and throw exeption, if invalid argument.